docs/02-app/01-building-your-application/02-data-fetching/03-forms-and-mutations.mdx

```diff
@@ -745,6 +780,17 @@ export async function create() {
 }
 ```
 
+```js filename="app/actions.js" switcher
+'use server'
+
+import { cookies } from 'next/headers'
+
+export async function create() {
+  const cart = await createCart():
```

Comment:
```suggestion
  const cart = await createCart()
```

Is the colon a typo?

I also see it on line 779 in the one above.

Reply:
Yes 👍